### ISSUE TITLE
Constraints Objects - Feature Request

### ISSUE THREAD

**Issue by nathanhack** [hamish2014/FreeCAD_assembly2#66]
Not trying to dictate an implementation, but once a part's label is updated constraints do not update their object names.  It makes it very difficult to figure out what the part is after updating labels.

**Comment by hamish2014** [comment]
Agreed, it can be frustrating determining which constraints belong to which parts.
Ultimately what is required is for the constraints to grouped under the parts see https://github.com/hamish2014/FreeCAD_assembly2/issues/36
